hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieROTablePathFilter.java

```diff
@@ -56,6 +58,15 @@
 public class HoodieROTablePathFilter implements Configurable, PathFilter, Serializable {
 
   private static final long serialVersionUID = 1L;
+  public static final String METADATA_PREFIX = "hoodie.metadata";
```

Comment:
minor: these seen to be duplicated from metadata code. If there is a simple way of moving them to a common class and accessing from there, that would be better.

Reply:
I have moved the `HoodieMetadataConfig` class to `hudi-common`.